### PR TITLE
Bug (Issue #56): Fix Error on dashboard 

### LINKED ISSUE #56: Save button not recognized after changing the date criteria
On the Dashboard Page, the Save chart button seems to not be working after the date criteria is changed. 
The user will get the following error message even after clicking the load chart button:

> Please load the chart by clicking on the 'Load Chart' button before saving it.

### Found By
- @cjayneb

### Proof
image

### PR DESCRIPTION
**Pull request by Funnyadd**
## Context:
This PR is related to the Issue #56 and fixes the bug described there.

## Changes
- Added a localstorage set for dash_previous_data in the chart function on the dashboard page.

## Linked Issue
- #56

### CODE REVIEW

File: client/src/pages/Dashboard.js [hunk 1]
```diff
@@ -328,6 +328,7 @@ const Dashboard = () => {
                     }
 
                     localStorage.setItem("dash_previous_chart_data", JSON.stringify(datasets));
+                    localStorage.setItem("dash_previous_criteria", JSON.stringify(criteria))
```
Comment (cjayneb) [inline client/src/pages/Dashboard.js:331]: Wait so this wasn't there before?

**Review by cjayneb — APPROVED** [review]
Im not sure how that line would've disappeared. But if the bug is gone, LGTM.

**Review by NickEspitalier — APPROVED** [review]
If the application works fine and that this bug is now gone. I have no problem with the current changes. LGTM

File: client/src/pages/Dashboard.js (hunk 1 above)
Comment (Funnyadd) [reply client/src/pages/Dashboard.js:331]: I think that by doing the previous feature, I changed the way data_previous_data was read and written by other parts of the code. I tested manually everything to make sure all the concerned features where working as intended. There might still be other issues that we are not aware of at the moment thought.

**Pull request merged**